### DISCUSSION TITLE
Updating Bookstack to 24.10.3 failed

### DISCUSSION THREAD

**Discussion by Hindin81** [community-scripts/ProxmoxVE#582] [Q&A] [answered]
When updating bookstack to the current version I got the following error: 
` ✓ Services Stopped
 / Updating Bookstack to 24.10.3   
[ERROR] in line 73: exit code 0: while executing command COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev &> /dev/null`

**Comment by Hindin81** [comment]
Can someone help me?

**Comment by michelroegl-brunner** [comment, marked as answer]
I´ve created a [PR](https://github.com/community-scripts/ProxmoxVE/pull/700) for this issue. When the maintainers accept it will get merged.
